plugins/package-managers/python/src/main/kotlin/utils/PythonInspector.kt

```diff
@@ -121,7 +123,9 @@ internal object PythonInspector : CommandLineTool {
                         "(${binaryResult.packages.size}), which might indicate a bug in python-inspector."
                 }
 
-                val resultsPurls = binaryResult.packages.mapTo(mutableSetOf()) { it.purl }
+                val resultsPurls = binaryResult.packages
+                    .mapNotNull { it.purl }
+                    .toMutableSet()
```

Comment:
Can be simplified to avoid creating an intermediary list: `binaryResult.packages.mapNotNullTo(mutableSetOf()) { it.purl }`

Reply:
Done